components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.viewport.js

```diff
@@ -1159,8 +1165,15 @@ jQuery._WeblitzViewport = function (container, server, options) {
     }
     if (query.maps) {
       query.maps.map(function(m, idx){
-        if (m.reverse) {
-          this.setChannelReverseIntensity(idx, m.reverse.enabled, true);
+        // Handle deprecated 'reverse' OR 'invert' codomain map
+        if (m.reverse || m.inverted) {
+          var enabled;
+          if (m.reverse) {
```

Comment:
This says to me that `reverse` has priority over `inverted` whereas above in `setActiveChannels` it's the other way around. Likely not a big issue, but if this is to be an example of deprecating a term, it might should be consistent.